pkg/version/version.go

```diff
@@ -26,7 +26,7 @@ const VersionPrefix = "v"
 
 // The current version of minishift
 // This is a private field and should be set when compiling with --ldflags="-X github.com/minishift/minishift/pkg/version.version=vX.Y.Z"
-var version = "v0.0.0-unset"
+var version = "v1.0.0-beta.5"
```

Comment:
-1, it is intentional to have this set to "unset". The version gets compiled in via the Makefile.